src/test/java/com/dataloom/datastore/authentication/MultipleAuthenticatedUsersBase.java

```diff
@@ -79,6 +81,7 @@ public static void loginAs( String user ) {
         requestsApi = currentRetrofit.create( RequestsApi.class );
         dataApi = currentRetrofit.create( DataApi.class );
         searchApi = currentRetrofit.create( SearchApi.class );
+        organizationsApi = currentRetrofit.create( OrganizationsApi.class );
```

Comment:
yes, there are some tests that extend `MultipleAuthenticatedUsersBase`. The updated `PermissionsControllerTest` needs `organizationsApi`.